server/src/com/cloud/storage/upload/UploadListener.java

```diff
@@ -270,7 +274,7 @@ public AgentControlAnswer processControlCommand(long agentId, AgentControlComman
     }
 
     public void setUploadInactive(Status reason) {
-        uploadActive = false;
+        // uploadActive = false;
```

Comment:
While making changes in this file, I noticed that Eclipse was pointing out that the variable uploadActive was only written to (never read from). Instead of removing the entire line, I just commented it out in case someone later wanted to easily resume making use of this variable.